src/ol3cesium.js

```diff
@@ -11,12 +11,26 @@ goog.require('olcs.VectorSynchronizer');
 
 
 /**
- * @param {!ol.Map} map
+ * @param {!(ol.Map|olcsx.OLCesiumOptions)} arg1 `ol.Map` or options
  * @param {Element|string=} opt_target Target element for the Cesium container.
+ *   Overrides a target set in options.
  * @constructor
  * @api
  */
-olcs.OLCesium = function(map, opt_target) {
+olcs.OLCesium = function(arg1, opt_target) {
```

Comment:
I do not think the way versioning is handled in ol3 is appropriate for
ol3cesium: ol3-cesium is very new compared to the openlayers project.

A better model for ol3-cesium is Cesium, which is a 2 year old project.